frontend/lib/src/components/widgets/Multiselect/Multiselect.tsx

```diff
@@ -370,7 +393,18 @@ const Multiselect: FC<Props> = props => {
                     : null,
               },
             },
-            Dropdown: { component: VirtualDropdown },
+            Dropdown: {
+              component: VirtualDropdown,
+              props: {
+                $menuListProps: {
+                  initialScrollOffset: getInitialScrollPosition(),
```

Comment:
What happens if the initialScrollOffset is larger as the available menu list (for some unexpected reason). Does it run into an error, or will it just scroll to the bottom?

Reply:
Just tested it, it simply scrolls to the end, doesn't break.

Reply:
ok 👍